sdk/mongocluster/arm-mongocluster/tsp-location.yaml

```diff
@@ -0,0 +1,3 @@
+directory: specification/mongocluster/DocumentDB.MongoCluster.Management
+commit: abad0096677005817d2c19df2364663e5583c8fc
+repo: Azure/azure-rest-api-specs
```

Comment:
I think we probably need to consider how to refresh the generated code with this, but it's probably not a major concern for now.